game_state.go

```diff
@@ -15,6 +15,7 @@ type GameState struct {
 	grenadeProjectiles map[int]*common.GrenadeProjectile // Maps entity-IDs to active nade-projectiles. That's grenades that have been thrown, but have not yet detonated.
 	infernos           map[int]*common.Inferno           // Maps entity-IDs to active infernos.
 	entities           map[int]*st.Entity                // Maps entity IDs to entities
+	bombState          *common.BombState
```

Comment:
I think it's nicer to take the reference when needed rather than storing it as pointer here.

Reply:
Excellent points and very nice catches w.r.t. code that was convoluted/not necessary!